netutils/ip.py

```diff
@@ -269,7 +271,7 @@ def cidr_to_netmaskv6(cidr):
     raise ValueError("Parameter must be an integer between 0 and 128.")
 
 
-def get_all_host(ip_network):
+def get_all_host(ip_network: str) -> t.List[str]:
```

Comment:
`-> Generator[str, None, None]` and then we can also get rid of the ignore comment